profilegen/src/res/templates/module_template.template

```diff
@@ -0,0 +1,37 @@
+# encoding: utf-8
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+require '<%= @module_require %>'
+
+module Azure
+  module Management
+    module Profiles
+      module <%= @profile_name %>
+        module <%= @module_name %>
+          #
+          # Class for the <%= @class_name %> module
```

Comment:
How about if we just say

```
#
# <%= @class_name %>
# 
```

Example we do in autorest: https://github.com/Azure/azure-sdk-for-ruby/blob/master/management/azure_mgmt_authorization/lib/generated/azure_mgmt_authorization/models/classic_administrator.rb#L9

Reply:
Done